examples/signer/example-signer.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"bufio"
 	"bytes"
+	crypto_rand "crypto/rand"
```

Comment:
I'm guessing this is to make it clear it isn't "math/rand" which is not cryptographically secure.
This is also consistent with how Curtis had used this elsewhere.

Reply:
I had to namespace it in examples/signer/example-signer.go for the reason @mlb7687 mentioned because we're using "math/rand" but I can change it in the other files.

Reply:
I think the explicit crypto_rand make sense. I was just curious.